### PR TITLE
Removed module description; Generation of config.json on publish

### PR DESCRIPTION
**Pull request by orlindraganov**

### CODE REVIEW

**Comment by Novkirishki** [comment]
No need to add ComponentsGenerated.wxs

File: Sitefinity CLI/Commands/AddModuleCommand.cs [hunk 1]
```diff
@@ -13,13 +13,6 @@ namespace Sitefinity_CLI.Commands
     [Command(Name = Constants.AddModuleCommandName, Description = "Adds a new module to the current project.", FullName = Constants.AddModuleCommandFullName)]
     internal class AddModuleCommand : AddToSolutionCommandBase
     {
-        /// <summary>
-        /// The description of the module, visible in Sitefinity / Modules and Services
-        /// </summary>
-        [Option(Constants.DescriptionOptionTemplate, Constants.DescriptionOptionDescription, CommandOptionType.SingleValue)]
-        [DefaultValue("")]
-        public string Description { get; set; }
-
```
Comment (Novkirishki) [inline Sitefinity CLI/Commands/AddModuleCommand.cs]: I thought that we will just remove description from the config file and therefore from the VSIX but not from the CLI. Is that what you agreed on with Vladi?

Comment (orlindraganov) [reply Sitefinity CLI/Commands/AddModuleCommand.cs]: We agreed the Description in the templates to be empty string along with an inline comment.

**Comment by Novkirishki** [comment]
Bump assembly version

**Pull request merged**